src/components/v2/appDetails/sourceInfo/environmentStatus/AppStatusDetailModal.tsx

```diff
@@ -177,50 +149,116 @@ function AppStatusDetailModal({ close, appStreamData, showAppStatusMessage }: Ap
                             {APP_STATUS_CUSTOM_MESSAGES[_appDetails.resourceTree.status.toUpperCase()]}
                         </div>
                     )}
-                    <div className="pt-16 pl-20 pb-8">
-                        <div className="flexbox pr-20 w-100">
-                            <div>
-                                <StatusFilterButtonComponent nodes={flattenedNodes} handleFilterClick={onFilterClick} />
-                            </div>
-                        </div>
-                    </div>
+                    <AppStatusDetailsChart appStreamData={appStreamData} />
+                </div>
+            </div>
+        </Drawer>
+    )
+}
+
+export function AppStatusDetailsChart({ appStreamData, filterRemoveHealth = false }) {
```

Comment:
Please extract out this as an individual function

Reply:
done